system-tests/004/004.test.js

```diff
@@ -0,0 +1,28 @@
+/* @flow */
+"use strict"
+const path = require("path")
+const spawn = require("child_process").spawn
+
+// this test ensures that CLI returns exit code which is not 0
+// when there is a runtime error in configuration javascript file
+// https://github.com/stylelint/stylelint/issues/2115
+
+it("004", (done) => {
+  const localPath = path.resolve(__dirname)
+  const cliPath = path.join(localPath, "../../lib/cli.js")
+
+  const cliProcess = spawn("node", [ cliPath, `${localPath}/*.css` ], { cwd: localPath })
+
+  cliProcess.on("error", function (error) {
+    console.log("error running cli:", error) // eslint-disable-line no-console
```

Comment:
it fails anyway if there is an error, it just added logging for that.
Couldn't reproduce a case when it didn't fail on error